staging/src/k8s.io/apiserver/pkg/util/peerproxy/peerproxy_handler_test.go

```diff
@@ -290,16 +353,18 @@ func newFakePeerProxyHandler(informerFinishedSync bool, reconciler reconcilers.P
 	}
 	ppI := NewPeerProxyHandler(informerFactory, storageversion.NewDefaultManager(), proxyRoundTripper, id, reconciler, s)
 	if testDataExists(svdata.gvr) {
-		ppI.addToStorageVersionMap(svdata.gvr, svdata.serverId)
+		ppI.addToStorageVersionMap(svdata.gvr, svdata.serverIds)
 	}
 	return ppI, nil
 }
 
-func (h *peerProxyHandler) addToStorageVersionMap(gvr schema.GroupVersionResource, serverId string) {
+func (h *peerProxyHandler) addToStorageVersionMap(gvr schema.GroupVersionResource, serverIds []string) {
 	apiserversi, _ := h.svMap.LoadOrStore(gvr, &sync.Map{})
 	apiservers := apiserversi.(*sync.Map)
-	if serverId != "" {
-		apiservers.Store(serverId, true)
+	for _, serverID := range serverIds {
```

Comment:
nit: Inconsistency casing for ID/Id

Reply:
Done

Reply:
Hahah linter failed after I modified it to be `id` instead of `ID` for consistency with other places. Guess have to make that change everywhere in the file.